Makefile

```diff
@@ -4,6 +4,8 @@ DUNE = opam exec -- dune
 
 .DEFAULT_GOAL := help
 
+DUNE_BUILD_DIR := $(shell pwd)/_build
```

Comment:
I set this so that the build directory is the same in CI even though no local switch is being used